src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/GitCommit.kt

```diff
@@ -1,3 +1,3 @@
 package org.vaccineimpact.orderlyweb.models
 
-data class GitCommit(val id: String, val dateTime: String, val displayDateTime: String)
\ No newline at end of file
+data class GitCommit(val id: String, val dateTime: String, val age: Int)
```

Comment:
Is this change to do with this ticket, or just an opportunisitc improvement?

Reply:
Turns out this wasn't being used in any tests - and actually not (any longer?) in sync with what is returned by server. Was necessary to fix so that I could use in the integration test [here](https://github.com/vimc/orderly-web/pull/283/files#diff-bd2bbf62dc816a76e47117557c906ea5f1e0ef25da9baec96e3b5245cc6a2715R64).